test/start-transcription-job.test.js

```diff
@@ -18,7 +18,6 @@ const args = {
 describe('start-transcription-job', () => {
   it('should start a transcription job if valid parameters supplied', async () => {
     const { data: transcribeJob, code } = await run('start-transcription-job', { args, config });
-    expect(code).to.equal(200);
```

Comment:
why remove this check? If the reason is valid you can remove `code` from the above const assignment

Reply:
This was not intentional. I probably deleted it during cleanup

Reply:
fixed